packages/ui/src/components/collectibles/native/inscription.native.tsx

```diff
@@ -5,28 +5,31 @@ import { CollectibleAudio } from './collectible-audio.native';
 import { CollectibleHtml } from './collectible-html.native';
 import { CollectibleImage } from './collectible-image.native';
 import { CollectibleText } from './collectible-text.native';
+import { CollectibleVideo } from './collectible-video.native';
 
 export interface InscriptionProps {
   mimeType: InscriptionMimeType;
   name: string;
   height: number;
   src: string;
+  onPress?: () => void;
 }
 
-export function Inscription({ mimeType, name, height = 200, src }: InscriptionProps) {
+export function Inscription({ mimeType, name, height = 200, src, onPress }: InscriptionProps) {
   switch (mimeType) {
     case 'audio':
-      return <CollectibleAudio size={height} />;
+      return <CollectibleAudio alt={name} src={src} size={height} onPress={onPress} />;
```

Comment:
I inscribed an audio ordinal and got it to render correctly however it times out at loading when trying to play it. I'll return to it